### PR TITLE
[TechDraw] Improve readability of APP scripts

### PR DESCRIPTION
**Pull request by benj5378**
Thank you for creating a pull request to contribute to FreeCAD! Place an "X" in between the brackets below to "check off" to confirm that you have satisfied the requirement, or ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10) if there is something you don't understand.

- [x]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

Please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [1.0 Changelog Forum Thread](https://forum.freecad.org/viewtopic.php?f=10&t=69438).

---

### CODE REVIEW

File: src/Mod/TechDraw/App/DrawLeaderLine.cpp [hunk 1]
```diff
@@ -139,141 +132,124 @@ App::DocumentObjectExecReturn *DrawLeaderLine::execute()
...
 double DrawLeaderLine::getBaseScale() const
 {
 //    Base::Console().Message("DLL::getBaseScale()\n");
-    double result = 1.0;
     DrawView* parent = getBaseView();
-    if (parent) {
-        result = parent->getScale();
+    if (!parent) {
+        return 1.0;
     }
```
Comment (benj5378) [inline src/Mod/TechDraw/App/DrawLeaderLine.cpp:166]:
```
if (!vp) { // :)
    return;
}
```

File: src/Mod/TechDraw/App/DrawLeaderLine.cpp [hunk 2]
```diff
@@ -107,20 +107,13 @@ void DrawLeaderLine::onChanged(const App::Property* prop)
...
 
     const App::DocumentObject* docObj = getBaseObject();
-    if (docObj) {
-        result = docObj->isTouched();                 //object property points to is touched
-    }
-    if (result) {
-        return result;
+    if (docObj && docObj->isTouched()) {
+        return true;  // Object property points to is touched
     }
```
Comment (benj5378) [inline src/Mod/TechDraw/App/DrawLeaderLine.cpp:117]: This script is probably the most weird I ever worked on. Here is just one example

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9187">pipeline status</a> for feature branch [PR_9187](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9187). Pipeline [828853886](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/828853886) was triggered at [6c6dba9](https://github.com/FreeCAD/FreeCAD/pull/9187/commits/6c6dba9722959af0531d8569a15f008060f5089e). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9187">pipeline status</a> for feature branch [PR_9187](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9187). Pipeline [828908388](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/828908388) was triggered at [8b135f0](https://github.com/FreeCAD/FreeCAD/pull/9187/commits/8b135f0cc7bbe5fa6b653ec06e120ae8c497a6ca). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by WandererFan** [comment]
A couple of general comments...
Are you using clang-format or are you doing everything by hand?  The reason I ask is that I think some of the reformatting you've done will get clobbered when the code goes through clang-format.   A specific example is making the long lines on multi-clause booleans into multiple lines.  I like your way better, but I think the clang-format file prefers the long line. :(

These changes are great and they do clean up the code, but they introduce lots of merge conflicts for anybody that has a big development effort underway.  That's why I would rather see readability changes done as part of a functional change.  A functional change is going to get tested harder than a reformat change, so there's a bit of risk avoidance too.

**Pull request merged**